tensorflow/compiler/xla/service/hlo_evaluator_typed_visitor.h

```diff
@@ -1008,7 +1008,8 @@ class HloEvaluatorTypedVisitor : public DfsHloVisitorWithDefault {
     return Status::OK();
   }
 
-  Status HandleConvolution(HloInstruction* conv) override {
+  template <typename InputT, typename AccumulatorT>
```

Comment:
Use InputT and AccumulateT instead of InputType and AccumulateType to match other type names in this file.